usedHttp.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:http/http.dart' as http;
+
+void makeRequest() async{
+    var response = await http.get('https://example.com/');
+    // If the http request is successful the statusCode will be 200
+    if (response.statusCode == 200){
+      var htmlToParse = response.body;
+      print(htmlToParse);
+    }
+}
+
+dynamic main(){
```

Comment:
Since main does not return a value, can you declare it as returning `void` instead of `dynamic`?

Reply:
I made a change as `void` instead of `dynamic`. Thank you for the amendment!